ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureCloudDestinationSync.java

```diff
@@ -837,93 +847,52 @@ public boolean doesBlobExist(BlobId blobId) {
   }
 
   @Override
-  public short updateBlobExpiration(BlobId blobId, long expirationTime, CloudUpdateValidator cloudUpdateValidator)
+  public short
```

Comment:
could you format this file?

Reply:
the latest version of intellij did this!